packages/nuxt-locokit/src/styles/primevue/_dialog.scss

```diff
@@ -21,44 +22,49 @@
 .p-dialog .p-dialog-header .p-dialog-header-icon {
   width: 2rem;
   height: 2rem;
-  color: #6c757d;
+  color: rgba(255, 255, 255, 0.6);
   border: 0 none;
   background: transparent;
   border-radius: 50%;
-  transition: background-color 0.2s, color 0.2s, box-shadow 0.2s;
+  transition: color 0.15s, box-shadow 0.15s;
   margin-right: 0.5rem;
 }
 
 .p-dialog .p-dialog-header .p-dialog-header-icon:enabled:hover {
   color: var(--text-color);
   border-color: transparent;
-  background: #e9ecef;
+  background: transparent;
 }
 
 .p-dialog .p-dialog-header .p-dialog-header-icon:focus {
   outline: 0 none;
   outline-offset: 0;
-  box-shadow: 0 0 0 0.2rem var(--primary-color-light);
+  box-shadow: 0 0 0 1px var(--primary-color-light);
 }
 
 .p-dialog .p-dialog-header .p-dialog-header-icon:last-child {
   margin-right: 0;
 }
 
 .p-dialog .p-dialog-content {
-  background: #ffffff;
+  background: #2a323d;
   color: var(--text-color);
-  padding: 0 1.5rem 2rem 1.5rem;
+  padding: 1rem;
+}
+
+.p-dialog .p-dialog-content:last-of-type {
+  border-bottom-right-radius: 4px;
+  border-bottom-left-radius: 4px;
 }
 
 .p-dialog .p-dialog-footer {
-  border-top: 0 none;
-  background: #ffffff;
+  border-top: 1px solid #3f4b5b;
+  background: #2a323d;
```

Comment:
same here

Reply:
Impossible to tell without testing the component. (cf https://github.com/locokit/locokit/issues/171)